packages/ckeditor5-dev-env/lib/release-tools/utils/hascommitsfromlastrelease.js

```diff
@@ -1,30 +0,0 @@
-/**
- * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
- * For licensing, see LICENSE.md.
- */
-
-'use strict';
-
-const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
-
-/**
- * Checks whether the package has committed changes until the last tag.
- * If the last tag does not exist, the function checks whether any commit exists.
- *
- * @returns {Boolean}
- */
-module.exports = function hasCommitsFromLastRelease() {
```

Comment:
It is not needed because [`getNewReleaseType()`]()https://github.com/ckeditor/ckeditor5-dev/pull/154/files#diff-81ddd240a176823f737da5a270374c71 knows all commits and can decide about the release type.